lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -138,7 +138,7 @@ export class EvaluationClientWrapper {
       const idRes = JsonPathUtils.extractInputField(pd, [idPath]);
       if (idRes.length) {
         const submissionRequirementMatch: SubmissionRequirementMatch = {
-          name: idRes[0].value.name,
+          name: idRes[0].value.id,
```

Comment:
Why is an id mapped to a name? This might cause future compatibility issues

Reply:
We can perhaps record both id and name. If name is not present at least we will have an id.

Reply:
Yes that makes more sense to me

Reply:
It seems that name/id could be anything, name from input descriptor or submission requirement, id from input descriptor or even pd.id if we have a from_nested use case. The is no consistent mapping, the use case is not clear to me so I decided to assign an uuid to that field to make it at least unique, just in case it meant to something in the future.